netam/dasm.py

```diff
@@ -195,11 +196,10 @@ def loss_of_batch(self, batch):
         # logit space, so we are set up for using the cross entropy loss.
         # However we have to mask out the sites that are not substituted, i.e.
         # the sites for which aa_subs_indicator is 0.
-        subs_mask = aa_subs_indicator == 1
+        subs_mask = (aa_subs_indicator == 1) & mask
```

Comment:
This is it, the whole Nan/inf fix! Could you have a look around here and make sure this looks reasonable to you, too?

Reply:
Nice sleuthing! It seems good to me... am I missing something?

Reply:
I don't think so, just wanted extra eyes on it! Thanks